File: Python PowerPoint Examples/MasterSlidesApiDemo/getMasterSlidesList.py
import spirecloudpowerpoint
from spirecloudpowerpoint.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey, baseUrl)
api = spirecloudpowerpoint.api.master_slides_api.MasterSlidesApi(configuration)

name = "GetMasterSlidesList.pptx"
password = ""
storage = ""
folder = "input"

result = api.get_master_slides_list(name=name,
                                    password=password, storage=storage,
                                    folder=folder)

File: PHP PowerPoint Examples/MasterSlidesApiDemo/getMasterSlidesList.php
<?php

use Spire\Cloud\Powerpoint\Sdk\Api\MasterSlidesApi;
use Spire\Cloud\Powerpoint\Sdk\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl="https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey,$baseUrl);
$apiInstance = new MasterSlidesApi($configuration);

$name = "GetMasterSlidesList.pptx";
$folder = "input";
$password = null;
$storage = null;
$result = $apiInstance->getMasterSlidesList($name, $password, $folder, $storage);
print_r($result);